src/cow-react/modules/limitOrders/hooks/useLimitOrdersFormState.ts

```diff
@@ -83,10 +87,11 @@ function getLimitOrdersFormState(params: LimitOrdersFormParams): LimitOrdersForm
   }
 
   if (
-    !inputCurrencyAmount ||
-    inputCurrencyAmount.toExact() === '0' ||
-    !outputCurrencyAmount ||
-    outputCurrencyAmount.toExact() === '0'
+    !isWrapOrUnwrap &&
+    (!inputCurrencyAmount ||
+      inputCurrencyAmount.toExact() === '0' ||
```

Comment:
We could use `!inputCurrencyAmount.equalTo(0)` instead

Reply:
Thanks! Fixed